pkg/controller/csi_handler.go

```diff
@@ -157,26 +157,44 @@ func (h *csiHandler) syncDetach(va *storage.VolumeAttachment) error {
 	return nil
 }
 
-func (h *csiHandler) addVAFinalizer(va *storage.VolumeAttachment) (*storage.VolumeAttachment, error) {
+func (h *csiHandler) prepareVAFinalizer(va *storage.VolumeAttachment) (*storage.VolumeAttachment, bool) {
```

Comment:
nit: name this bool return var or add a comment as description

Reply:
ack